module2/assignment2.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Cook | INFO 2439 | Assignment 2</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="_css/Cook-a2.css">
```

Comment:
I'd consider listing the jquery css first so that your custom css takes higher priority.
That way jquery isn't always overwriting your custom styles.